app/forms/sections/info/LocalReferenceNumberFormProvider.scala

```diff
@@ -17,17 +17,23 @@
 package forms.sections.info
 
 import forms.mappings.Mappings
+import models.requests.DataRequest
+import pages.sections.info.LocalReferenceNumberPage
 import play.api.data.Form
 
 import javax.inject.Inject
 
 class LocalReferenceNumberFormProvider @Inject() extends Mappings {
 
-  def apply(isDeferred: Boolean): Form[String] =
+  def apply(isDeferred: Boolean)(implicit dataRequest: DataRequest[_]): Form[String] = {
+    val optOriginalValueSentInPreviousSubmission = dataRequest.userAnswers.getOriginalAttributeValueForPage(LocalReferenceNumberPage())
     Form(
       "value" -> text(errMsgForKey("required")(isDeferred))
         .verifying(maxLength(22, errMsgForKey("length")(isDeferred)))
+        //TODO: can getOrElse be changed to something else?
```

Comment:
(Not tested, but) could you flip the condition check to something like this:
```scala
protected def isNotEqualToOpt(existingAnswer: Option[String], errorKey: String): Constraint[String] =
  Constraint {
    case answer if existingAnswer.contains(answer) =>
      Invalid(errorKey)
    case _ =>
      Valid
  }
```

then use it like `.verifying(isNotEqualToOpt(optOriginalValueSentInPreviousSubmission, "errors.704.lrn.input"))`

?